org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/parser/JavadocTagConstants.java

```diff
@@ -180,101 +181,54 @@ public interface JavadocTagConstants {
 
 	// href tag
 	public final static char[] HREF_TAG = {'h', 'r', 'e', 'f'};
+
+	public static class LevelTags {
```

Comment:
A javadoc explaining
1) purpose
2) structure
3) update policy

is needed here.
The old code didn't provided that, but this shouldn't be an excuse.

Reply:
Nothing to sorry. Just imagine someone would need to update this code in few years to make Java 42 compatible.

Write docs/instructions for this case, assuming it would be not you doing the change. That's all what I expect to be added. 

Note, I also have no clue about this code, and wondered few times before what all these arrays are supposed to do.